components/threading/global_threads/thread_item/thread_item.tsx

```diff
@@ -186,17 +189,22 @@ function ThreadItem({
                         )}
                     </div>
                 )}
-                <span>{postAuthor}</span>
-                {Boolean(channel) && (
-                    <Badge
-                        className={classNames({
-                            Badge__hidden: postAuthor === channel?.display_name,
-                        })}
-                        onClick={goToInChannelHandler}
-                    >
-                        {channel?.display_name}
-                    </Badge>
-                )}
+                <div className='ThreadItem__author'>{postAuthor}</div>
+                <div className='d-flex align-items-center'>
+                    {Boolean(channel) && postAuthor !== channel?.display_name && (
```

Comment:
no need for an extra `Boolean` cast
```suggestion
                    {channel && postAuthor !== channel?.display_name && (
```